content/en/docs/concepts/scheduling-eviction/scheduling-framework.md

```diff
@@ -11,27 +11,26 @@ weight: 60
 {{< feature-state for_k8s_version="v1.19" state="stable" >}}
 
 The scheduling framework is a pluggable architecture for the Kubernetes scheduler.
-It adds a new set of "plugin" APIs to the existing scheduler. Plugins are compiled into the scheduler. The APIs allow most scheduling features to be implemented as plugins, while keeping the
-scheduling "core" lightweight and maintainable. Refer to the [design proposal of the
-scheduling framework][kep] for more technical information on the design of the
-framework.
+It adds a new set of "plugin" APIs to the existing scheduler. Plugins are compiled
```

Comment:
Ideally, also rewrite this to be [timeless](https://developers.google.com/style/timeless-documentation): avoid “new”, “existing”.